feat: Troubleshooting page for Capacitor with iOS Deployment Version fix (#3839)
getsentry/sentry-docs: src/platforms/javascript/guides/capacitor/troubleshooting.mdx

---
title: Troubleshooting

> description: "Troubleshoot and resolve common issues with the Capacitor SDK."

Reviewer: You might also add keywords to help with search, like this:
keywords: ["minimum version"]

sidebar_order: 1000
---

> ## Using with Capacitor 2 on iOS

Reviewer 2: ```suggestion
## Capacitor 2 on iOS
```

> Because Capacitor 3 has a minimum requirement of iOS 12.0, the Sentry SDK needs to match this requirement to support Capacitor 3. Sadly, users on older versions my run into an error such as

Reviewer 2: ```suggestion
Because Capacitor 3 has a minimum requirement of iOS 12.0, the Sentry SDK needs to match this requirement to support Capacitor 3. Users on older versions may run into the following error:
```

> Because Capacitor 3 has a minimum requirement of iOS 12.0, the Sentry SDK needs to match this requirement to support Capacitor 3. Sadly, users on older versions my run into an error such as

Reviewer: Minor edit to Philipp's edit:
```suggestion
Capacitor 3 has a minimum requirement of iOS 12.0. As a result, the Sentry SDK needs to match this requirement to support Capacitor 3. Users on older versions may run into an error similar to:
```

> Specs satisfying the SentryCapacitor (from ../../node_modules/@sentry/capacitor) dependency were found, but they required a higher minimum deployment target.

> You can fix this by bumping your iOS deployment target up. Add the below snippet to your `capacitor.config.json`:

Reviewer 2: ```suggestion
You can fix this by bumping your iOS deployment target. Add the snippet below to your `capacitor.config.json`:
```
